ast/src/parsed/asm.rs

```diff
@@ -394,6 +394,7 @@ impl<T: Clone> Machine<T> {
 pub struct MachineArguments {
     pub latch: Option<String>,
     pub operation_id: Option<String>,
+    pub selector: Option<String>,
```

Comment:
I think this has to be added to `Machine::local_names` if it is a symbol declared by the machine statement that should be accessible inside the machine.